macleann/enchiridion-client
# Adding likes functionality

# Implementing 'Likes' Feature for Playlists

This is something I've wanted to do for some time! Glad I'm finally getting around to it. There's definitely more to clean up about likes and all, but it's fine so far. A decent amount of styling had to go into this push so that's where a bulk of the changes here are coming from. I did DRY up some of my code with regards to mapping through playlists on the home page and the playlists page.

### Bug fixes:
Fixes #36 Feature: Users can like playlists
Fixes #37 Feature: Most Liked playlists is displayed on Home tab

## Type of change


# How to Test?

Make sure you've followed all installation and setup instructions in the README. Requires pulling the most recent `main` branch of [enchiridion-server](https://github.com/macleann/enchiridion-server).

```
git fetch origin nm-likes
git checkout nm-likes
npm start
```

- [ ] Ensure you are logged in to test the new 'like' feature.
- [ ] Navigate to the playlist page to see the new 'like' button.
- [ ] Click the 'like' button and ensure the count increments.
- [ ] Refresh the page to ensure the 'like' persists.

# Checklist:
